### ISSUE TITLE
Error in `beir/beir/retrieval/search/sparse/sparse_search.py`

### ISSUE THREAD

**Issue by jin530** [UKPLab/beir#60]
https://github.com/UKPLab/beir/blob/568f4c34fa0be1901e2aaa8479978c9e54a1e377/beir/retrieval/search/sparse/sparse_search.py#L28-L34

Hello, 

I think there is an error in `sparse_search.py`; only the first query in each batch is evaluated. 

Thank you!

**Comment by maximedb** [comment]
Indeed! An easy fix would be to remove the step size `self.batch_size`?
